models/clm/tutorial/README.rst

```diff
@@ -1728,6 +1741,18 @@ scatterplot compares the prior expected observations vs. the actual observations
 The 1:1 fit for this plot is poor, but should be slightly improved if you
 compare the posterior observations vs. the actual observations.
 
+.. Note::
+
+ The matlab geographic scatterplot is rendered in 3D and can be converted into 2D
+ (as it appears below) by using the 'Rotate 3D' option at the
+ top of the figure or through the menu bar as Tools > Rotate 3D. Use the cursor to 
+ rotate the map such that the vertical dimension is removed. In the case of the
+ MODIS LEAF AREA observation type no vertical value is specified as the vertical code = -2
+ which is 'undefined'.  Thus the ``-888888`` value for the ``vertical level`` acts as a 
+ missing  value that DART ignores, and has no impact on the assimilation. For more information about
+ specifying vertical heights in observations go 
+ :doc:`here. <../../../guide/creating-obs-seq-real>`  
```

Comment:
I think this paragraph could be made a bit clearer. I've put a suggestion below which uses 'coordinate' for the vertical measurement.  coordinate is the term used in the  location modules for the vertical (height, pressure, level, etc.)

I also changed the link so it uses the heading rather than 'here'


```suggestion
rotate the map such that the vertical dimension is removed.  For 3D observations with no vertical coordinate, such as MODIS LEAF AREA, DART sets ``VERTISUNDEF`` for the vertical coordinate and ``-888888`` as the vertical value.  During assimilation, DART ignores the missing vertical dimension for observations with ``VERTISUNDEF``. For more information about
 specifying vertical coordinates for observations see 
 :doc:`../../../guide/creating-obs-seq-real`.
```